eldritch/test_events.py

```diff
@@ -822,6 +822,56 @@ def testCombatChoice(self):
 
 # TODO: add tests for going unconscious/insane during a mythos/encounter.
 
+class CloseLocationTest(EventTest):
+  def testCloseForever(self):
+    place_name = "Woods"
+    self.state.event_stack.append(CloseLocation(place_name))
+    self.resolve_until_done()
+    self.assertTrue(self.state.places[place_name].closed)
```

Comment:
Maybe advance one or two turns and verify that the location is still closed?

Reply:
Roger